frontend/src/pages/AllMovies.jsx

```diff
@@ -9,17 +10,131 @@ function AllMovies({ sortType }) {
+          </Select>
+
+          <Stack width="80%" spacing={[1, 5]} direction={["column"]}>
+            <Checkbox
+              size="lg"
+              colorScheme="teal"
+              defaultChecked
+              display="flex"
+              align="center"
+            >
+              <Box display="flex" alignItems="center">
+                {Array(5)
+                  .fill("")
+                  .map((_, i) => (
+                    <StarIcon
+                      w={4}
+                      h={4}
+                      color={i < 5 ? "teal.500" : "gray.300"}
+                    />
+                  ))}
+              </Box>
+            </Checkbox>
+            <Checkbox size="lg" colorScheme="teal" defaultChecked>
+              <Box display="flex" alignItems="center">
+                {Array(5)
+                  .fill("")
+                  .map((_, i) => (
+                    <StarIcon
+                      w={4}
+                      h={4}
+                      color={i < 4 ? "teal.500" : "gray.300"}
+                    />
+                  ))}
+              </Box>
+            </Checkbox>
+            <Checkbox size="lg" colorScheme="teal" defaultChecked>
+              <Box display="flex" alignItems="center">
+                {Array(5)
+                  .fill("")
+                  .map((_, i) => (
+                    <StarIcon
+                      w={4}
+                      h={4}
+                      color={i < 3 ? "teal.500" : "gray.300"}
+                    />
+                  ))}
+              </Box>{" "}
+            </Checkbox>
+            <Checkbox size="lg" colorScheme="teal" defaultChecked>
+              <Box display="flex" alignItems="center">
+                {Array(5)
+                  .fill("")
+                  .map((_, i) => (
+                    <StarIcon
+                      w={4}
+                      h={4}
+                      color={i < 2 ? "teal.500" : "gray.300"}
+                    />
+                  ))}
+              </Box>{" "}
+            </Checkbox>
+            <Checkbox size="lg" colorScheme="teal" defaultChecked>
+              <Box display="flex" alignItems="center">
+                {Array(5)
+                  .fill("")
+                  .map((_, i) => (
+                    <StarIcon
+                      w={4}
+                      h={4}
+                      color={i < 1 ? "teal.500" : "gray.300"}
+                    />
+                  ))}
+              </Box>{" "}
+            </Checkbox>
+            <Checkbox size="lg" colorScheme="teal" defaultChecked>
+              <Box display="flex" alignItems="center">
+                {Array(5)
+                  .fill("")
+                  .map((_, i) => (
+                    <StarIcon
+                      w={4}
+                      h={4}
+                      color={i < 0 ? "teal.500" : "gray.300"}
+                    />
+                  ))}
```

Comment:
Too many repetitions, there must be a way to manage this a better way 😉

Reply:
ok this was ugly ^^ but fixed now!